### DISCUSSION TITLE
Report Management

### DISCUSSION THREAD

**Discussion by MStahl15** [GoogleChrome/lighthouse#13052] [General] [answered]
Hypothetical:
I have an application that has 5 pages.  I run Lighthouse on each page generating 5 reports. (1 per page)   

Question:
Is there a way to view or consolidate the 5 .html reports into a single .html report that would allow for interacting with individually selectable page reports?

**Comment by adamraine** [comment, marked as answer]
This is being worked on right now, it's part of #11313.

**Reply by MStahl15** [reply]
Thanks!
